scripts/linters/general_purpose_linter.py

```diff
@@ -547,7 +546,7 @@ def check_extra_js_files(self):
                     ('core/templates', 'extensions')) and (
                         filepath not in build.JS_FILEPATHS_NOT_TO_BUILD
                         ) and not filepath.endswith(
-                        ('protractor.js', 'webdriverio.js')):
+                        ('webdriverio.js')):
```

Comment:
You don't need the parentheses anymore

```suggestion
                        'webdriverio.js'):
```

Reply:
Also, you can probably move this to the previous line.

Reply:
Done.